src/main/java/com/redhat/devtools/intellij/quarkus/buildtool/maven/MavenToolDelegate.java

```diff
@@ -93,8 +93,9 @@ public void processImport(Module module) {
             var distributionUrl = getWrapperDistributionUrl(ProjectUtil.guessProjectDir(project));
             if (distributionUrl != null) {
                 String mavenHome = mavenSettings.getMavenHome();
-                if (!MavenServerManager.WRAPPED_MAVEN.equals(mavenHome)){
-                    mavenSettings.setMavenHome(MavenServerManager.WRAPPED_MAVEN);
+                String WRAPPED_MAVEN = "Use Maven wrapper"; //MavenServerManager.WRAPPED_MAVEN was removed in 2023.3 without any deprecation warning!
```

Comment:
define a private static field

Reply:
that's what I did 1st, but intellij said it was better to keep it local

Reply:
very strange? Ok as you wish.